Microsoft.DotNet.Interactive/Commands/LoadExtensionsInDirectory.cs

```diff
@@ -11,15 +11,15 @@ namespace Microsoft.DotNet.Interactive.Commands
     public class LoadExtensionsInDirectory : KernelCommandBase
     {
         public LoadExtensionsInDirectory(
-            DirectoryInfo directory, 
-            IReadOnlyList<FileInfo> additionalDependencies = null)
+            string directoryPath, 
```

Comment:
I'd prefer not to weaken the type safety here. What's the reasoning?

Reply:
DirectoryInfo and FileInfo do not serialize by newtonsoft json.